### PR TITLE
Copy Editing: Eradicate the usage of informal `you'; clean up surrounding text

### PR DESCRIPTION
**Pull request by mfwitten**
(Ultimately, this commit should not change the meaning of the text.)

Because this is a standard, it is probably best not to employ the rather
informal second person, `you'. This commit removes that usage without
descending into overtly bureaucratic language.

Furthermore, as a matter of practicality or even necessity, some of
the nearby text has also been improved (according to this author);
this includes rewording, re-styling, the removal of typos, and the
fixing of grammar.

### CODE REVIEW

File: master/shapes.html [hunk 1]
```diff
@@ -256,7 +256,7 @@ <h2 id="EllipseElement">The <span class="element-name">'ellipse'</span> element<
 <p>
   The <a>'cx'</a> and <a>'cy'</a> coordinates define the center of the ellipse.
 </p>
-
+</bx
```
Comment (SebastianZ) [inline master/shapes.html]: This is obviously a syntax error.

Sebastian

Comment (mfwitten) [inline master/shapes.html]: *Woops!* How embarrassing. I must have introduced that error at the very last moment; clearly, I need to figure out how to introduce an XML validator into my workflow.

As this is an obvious and trivial mistake, I've force-pushed an amended commit.

Thanks!

File: master/embedded.html [hunk 2]
```diff
@@ -457,17 +457,19 @@ <h2 id="HTMLElements">HTML elements in SVG subtrees</h2>
...
-  Therefore, in order to include these elements within your SVG, 
-  you must either use an XML serialization that recognizes namespace declarations 
-  (such as stand-alone SVG or XHTML),
-  or you must create namespaced elements via the <code>createElementNS</code> DOM API method. 
-  Alternatively, you can use the foreignObject element 
-  to wrap the HTML-namespaced elements; 
-  the HTML parser correctly parses HTML elements within a foreignObject.
-</p>
+<div class="note">
+  <p class="note">
```
Comment (mfwitten) [inline master/embedded.html]: I should probably remove this unnecessary `class="note"`, as it is already specified by the parent element.

**Comment by nikosandronikos** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Comment by nikosandronikos** [comment]
This improves the text in many places. Thanks!

**Pull request merged**